### PR TITLE
Add `disable-focus` Attribute to Prevent Focus on Menu Open

### PR DESCRIPTION
**Pull request by scottosmith**
# Developer Notes

- Adds ability to disable the focus on the menu when it's opened (not ideal for accessibility reasons!) via the `disable-focus` attribute. Defaults to`false`. 

- `tabindex` is now bound so it can be set in `{{mdc-menu}}`

### CODE REVIEW

File: addon/components/mdc-menu.js [hunk 1]
```diff
@@ -28,6 +28,11 @@ export default Ember.Component.extend(MDCComponent, {
    * @type {?Function}
    */
   cancel: x => x,
+  /**
+   * Enabling this is generally a bad idea for accessibility, so think hard about using this attribute.
```
Comment (danielraggs) [inline addon/components/mdc-menu.js]: This comment seems a little awkward for what the attribute is called. When you say "enabling this," you mean "disabling focus" which seems a little backwards to me. :shrug:

Comment (danielraggs) [reply addon/components/mdc-menu.js]: I guess I'd recommend changing the comment to say "Disabling focus is generally a bad idea [...]"

Comment (scottosmith) [reply addon/components/mdc-menu.js]: Enabling = setting this to `true` = what would be generally bad for accessibility

Comment (danielraggs) [reply addon/components/mdc-menu.js]: yeah but setting to true means disabling. so you're saying enabling the disabling is bad for accessibility. Idk, just sounds weird to me but that could just be me.

Comment (Kerrick) [reply addon/components/mdc-menu.js]: I agree that using the word "enabling" to talk about disabling could be confusing. Better safe (explicit) than sorry (causing misunderstandings that lead to bugs)!

**Pull request merged**